### ISSUE TITLE
Files have a different filename if synchronized with the scoped app feature

### ISSUE THREAD

**Issue by Blenderpics** [arnoudkooi/sn-scriptsync#18]
If you synchronize a script containing whitespaces with the "scoped app" feature, the filename in the FS will contain those whitespaces.  
if you synchronize the same script directly using the script field save icon, the whitespaces will get replaced with dashes.

**Comment by Blenderpics** [comment]
I will take a look at it. if i find some time, i will fix it myself

**Comment by arnoudkooi** [comment]
great, thanks for opening the issue

**Comment by arnoudkooi** [comment]
Should be fixed in 1.5.6 please confirm

**Comment by Blenderpics** [comment]
I will check if it's fixed on monday.  
I already investigated this a bit, but it seems you were faster ;)  
Thank!

**Comment by Blenderpics** [comment]
Sorry for the huge delay.  
I have a lot going on lately.  
I checked the behaviour with the current version of sn-scriptsync.  
The issue is still present. I checked the git history and found your commit 619618ad3d3ad6a628a2d57c095813f709520975.  
The `.replace(/\./g, '-')` you have added only replaces dots with dashes. The issue with spaces still remain.  
I'm not even sure if this is really a bug of sn-scriptsync. If you click on the sync button in the servicenow studio, sn-scriptsync receives a file list. In this file list, the names are already different compared to the the normal sync method.  
I guess SN Utils is doing some kind of normalization at some point for the normal sync functionality, but not for the scoped app sync
